ivy/functional/frontends/paddle/tensor/logic.py

```diff
@@ -75,3 +75,11 @@ def less_than(x, y, /, *, name=None):
 @to_ivy_arrays_and_back
 def less_equal(x, y, /, *, name=None):
     return ivy.less_equal(x, y)
+
+@with_unsupported_dtypes(
+    {"2.4.2 and below": ("bool", "uint8", "int8", "int16", "complex64", "complex128")},
```

Comment:
Please update the supported dtypes according to the paddle's doc - https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/logical_and_en.html

Reply:
Made the changes. Please review.